src/figanos/matplotlib/utils.py

```diff
@@ -968,10 +968,33 @@ def get_rotpole(xr_obj: xr.DataArray | xr.Dataset) -> ccrs.RotatedPole | None:
     ccrs.RotatedPole or None
     """
     try:
+
+        if isinstance(xr_obj, xr.Dataset):
+            ds = xr_obj
+            gridmap = [
+                ds[v].attrs["grid_mapping"]
+                for v in ds.data_vars
+                if "grid_mapping" in ds[v].attrs
+            ]
+            gridmap += [
+                c for c in ds.coords if ds[c].attrs.get("grid_mapping_name", None)
+            ]
+            gridmap = list(np.unique(gridmap))
```

Comment:
La fonction de xscen regarde tous les grid mappings possibles. Comme ici on sait d'avance qu'on cherche un rotated pole, on peut simplifier il me semble :

```suggestion
            gridmap = xr_obj.cf.grid_mapping_names.get('rotated_latitude_longitude', [])
```

Reply:
Oui! La liste des possibles est ici : https://cfconventions.org/Data/cf-conventions/cf-conventions-1.12/cf-conventions.html#appendix-grid-mappings

Reply:
merci!